common/src/jni/main/include/conscrypt/logging.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CONSCRYPT_LOGGING_H_
+#define CONSCRYPT_LOGGING_H_
+
+#include <conscrypt/macros.h>
+
+#define LOG_TAG "NativeCrypto"
+
+#ifndef CONSCRYPT_UNBUNDLED
```

Comment:
I'm confused as to when `CONSCRYPT_UNBUNDLED` is set. I thought it was just for unbundled _Android_ builds. If so, doesn't this block also apply to OpenJDK builds?

Reply:
It's set in every build that isn't an Android platform build, including OpenJDK builds.  "Unbundled" here means "not bundled with the Android OS".

Reply:
OK, that makes sense.

It looks like `CONSCRYPT_UNBUNDLED` is set whenever `CONSCRYPT_NOT_UNBUNDLED` is not _exlicitly_ set (in `macros.h`). That seems correct, but it only happens if `JNI_JARJAR_PREFIX` is also not set. Is that intentional?

Reply:
Yeah, it's a weird setup, and probably could be set up more obviously.  We have three builds:

Normal open source build: Uses Gradle, no definitions.  CONSCRYPT_UNBUNDLED is set.

GMS builds: Uses a build system not exposed here, sets -DJNI_JARJAR_PREFIX and -DCONSCRYPT_UNBUNDLED.  CONSCRYPT_UNBUNDLED is set.

Android platform: Uses Android.bp, sets only -DJNI_JARJAR_PREFIX.  CONSCRYPT_UNBUNDLED is unset.